tests/test_utilities.py

```diff
@@ -778,3 +778,22 @@ def test_has_duplicates():
 
     with pytest.raises(ValueError):
         raise_has_duplicates(np.array([0, 1, 2, 2]))
+
+
+def test_copy_vtk_array():
+    with pytest.raises(TypeError, match='Invalid type'):
+        pyvista.utilities.misc.copy_vtk_array([1, 2, 3])
+
+    value = 10
+    arr = vtk.vtkFloatArray()
+    arr.SetNumberOfValues(2)
+    arr.SetValue(0, value)
+    arr_copy = pyvista.utilities.misc.copy_vtk_array(arr, deep=True)
+    arr_copy.GetValue(0)
+    assert value == arr_copy.GetValue(0)
+
+    arr_copy_not_deep = pyvista.utilities.misc.copy_vtk_array(arr, deep=False)
+    new_value = 5
+    arr.SetValue(1, new_value)
+    arr_copy.GetValue(1)
```

Comment:
Similar to the above, not sure what this line tests. The original (deep) copy doesn't have any value set for index 1, so this result would be undefined I think. What we might want to test is that the old deep copy didn't change, but for that we have to initialise index 1 before the first copy.

Reply:
Resolved in 129477c3c4867152eb1dfce0f50730aa61c3bc0c.

Reply:
I think this line survived the cleanup by accident. There's still a throwaway `GetValue(1)` line, instead I'd expect something like `assert value == arr_copy.GetValue(1)`

Reply:
Quite right.

```suggestion
    assert value == arr_copy.GetValue(1)
```